src/linagora.esn.resource/app/app.js

```diff
@@ -12,3 +12,37 @@
     'restangular'
   ]);
 })(angular);
+
+require('esn-frontend-common-libs/src/frontend/js/modules/attendee/attendee.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/user/user.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/session.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/lodash-wrapper.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/async-action.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/infinite-list/infinite-list.module.js');
+
```

Comment:
missing require for the `notificationFactory` and `esnI18nService` and other core services like `emailService`, should be
```js
require('esn-frontend-common-libs/src/frontend/js/modules/notification.js');
require('esn-frontend-common-libs/src/frontend/js/modules/i18n/i18n.module.js');
require('esn-frontend-common-libs/src/frontend/js/modules/core.js');
```